### DISCUSSION TITLE
CSV Standard to upload old data

### DISCUSSION THREAD

**Discussion by kaciker** [AlfHou/hammond#32] [General]
I never achivied to upload data from csv to the project. I was asking for a standard csv that works, and all that I saw was some examples that for me never worked. I have my data in a app "Ecodrive" that allow to me to export. I'm sure that I can adapt my data to the standard csv.
I'm using this csv, creating the car name "My Camry" Hammond give me the error but not specifying which one or where is the problem
<img width="984" alt="image" src="https://user-images.githubusercontent.com/7668181/216035089-58ccd248-93b1-4f42-ab05-1d1b14d429e9.png">

Could someone provide to me a CSV that works to be able to follow it ?? Thanks to support this project 

[Fuelly.Export.csv](https://github.com/AlfHou/hammond/files/10556594/Fuelly.Export.csv)

**Comment by kaciker** [comment]
I think I have "almost" got a master file for this, but I have a problem with the decimals of the price, I have had a issue, here #33

**Comment by AlfHou** [comment]
I'll take a look at fixing the decimal issues. If you share the direct export from the Ecodrive app, I'll take a look at creating a custom importer for it as well. Might make the friction less for other people migrating as well :)

**Reply by kaciker** [reply]
that could be fantastic ¡¡¡

Attached one of my files.
[Blue  Shark_.csv](https://github.com/AlfHou/hammond/files/10567589/Blue.Shark_.csv)
This is very simple app, and eassy to export, but I want to strat to use Hammond asasp.

<img width="353" alt="image" src="https://user-images.githubusercontent.com/7668181/216293166-fa069499-31e8-431a-b4ce-31b2f42ef3f0.png">
<img width="359" alt="image" src="https://user-images.githubusercontent.com/7668181/216293260-1c3c337a-625e-44d8-9439-c4df2733478c.png">

**Comment by kaciker** [comment]
BTW... looking for solutions, I discovered this site, that convert files from Fuel, aCar, Drivvo and Motostat, to Fuelio's format. Maybe adding Fuelio to the options integration, you can multiply by 3 the optioons os ussability 

https://fuelio.programistyk.pl/
